### ISSUE TITLE
Array items error message not comprehensible

### ISSUE THREAD

**Issue by crevete** [nicolasblanco/rails_param#84]
## Basic Info

* rails_param Version: 1.1
* Ruby Version: 2.6.6
* Rails Version: 5.2.6

## Issue description

When using Array validator, the error message is incomprehensible:
```ruby
param! :status, Array do |array, index|
  array.param! index, String, in: %w[a b c]
end
```
The error message is `Parameter 0 must be within [\"a\", \"b\", \"c\"]`

I expect to receive, at least, something with an explicit context, like `Parameter status[0] must be within [\"a\", \"b\", \"c\"]`.
Imagine that when I have several Array parameters to be validated, I can never know from which one the error comes...

Thanks

**Comment by ifellinaholeonce** [comment]
Thanks for raising this issue. I can see that being a feature others would benefit from as well. If you would like to open a Pull Request to add this functionality I would be happy to review it.

**Comment by iMacTia** [comment]
Closed by #85
